FormBuilder.php

```diff
@@ -905,7 +905,7 @@ protected function getModelValueAttribute($name)
 	{
 		if (is_object($this->model))
 		{
-			return object_get($this->model, $this->transformKey($name));
+			return data_get($this->model, $this->transformKey($name));
```

Comment:
Could you not just simply return `data_get` from the function as a whole and skip the whole `is_object` / `is_array` check because data_get does all that internally anyway. Would simplify things and improve support overall...